app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py

```diff
@@ -71,16 +82,18 @@ async def raster_tile_cache_asset(
                 "source_type": RasterSourceType.raster,
                 "source_driver": RasterDrivers.geotiff,
                 "source_uri": new_source_uri,
-                "calc": None,
+                "calc": calc,
                 "resampling": resampling,
                 "compute_stats": False,
                 "compute_histogram": False,
                 "symbology": Symbology(**symbology),
+                "subset": None,
             },
         }
     )
 
     # If float data type, convert to int in derivative assets for performance
+    # FIXME: Make this work for multi-band inputs
```

Comment:
did you mean to do this now or later?

Reply:
Later. Single-band float datasets will continue to work as before. Do we have any multi-band float datasets on the horizon @manukala6 ?